### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by anoopjb**
- Async binding operations
- GET Endpoints

### CODE REVIEW

File: docs/Interoperator-Features.md [hunk 1]
```diff
@@ -78,4 +78,21 @@ As a Service Provider, if one wants to define quotas on services provisioned, It
...
+  * Remaining quota is calculated based on above two values and based on that provision request is allowed.
+
+## Asynchronous Service Binding Operations
+
+If the service supports, it is possible to perform service binding operations asynchronously. To leverage asynchronous service binding for the plans, under SFPlans, one needs to add the following metadata.
+
+```
+spec
+  manager:
+  async: true
```
Comment (vivekzhere) [inline docs/Interoperator-Features.md:90]: Indentation

File: docs/Interoperator-Features.md [hunk 2]
```diff
@@ -78,4 +78,21 @@ As a Service Provider, if one wants to define quotas on services provisioned, It
...
+  asyncBinding: false
+
+```
+
+If `asyncBinding` is set to `true` for the plan, the service binding operation will be asynchronous. The default value is `false` if omitted from the plan and the default behavior is synchronous. The `async` flag is used to control the behavior of service provisioning operations.
```
Comment (vivekzhere) [inline docs/Interoperator-Features.md:95]: will be asynchronous if the platform passes the `accept_incomplete` flag as `true`

**Review by vivekzhere — APPROVED** [review]
minor comments

File: docs/Interoperator-Features.md (hunk 1 above)
Comment (anoopjb) [reply docs/Interoperator-Features.md]: Updated.

File: docs/Interoperator-Features.md (hunk 2 above)
Comment (anoopjb) [reply docs/Interoperator-Features.md]: Added.

**Review by vivekzhere — APPROVED** [review]

**Pull request merged**